src/data/sql-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const sqlQuiz = [
+  {
+    Question: "In SQL, what does CRUD Operations stand for?",
```

Comment:
it will sound better grammatically to drop the word operations

```suggestion
    Question: "In SQL, what does CRUD stand for?",
```